controllers/host/platformnetworks.go

```diff
@@ -809,6 +841,14 @@ func (r *HostReconciler) ReconcileNetworkAndAddressPools(
 		is_reconciled,
 		*should_reconcile)
 
+	if *should_reconcile && err != nil {
```

Comment:
Can we change the "should_reconcile" to "reconcile_expected"?

Reply:
Done.